packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -51,6 +51,8 @@ const AppConfig = ({children, locals = {}}) => {
     }
 
     const commerceApiConfig = locals.appConfig.commerceAPI
+    const passwordlessConfig = locals.appConfig.login.passwordless
+
```

Comment:
Read passwordless login config here and pass to `<CommerceApiProvider/>` to read either the `callbackURI` or `mode`